.gitignore

```diff
@@ -4,6 +4,7 @@
 *.dll
 *.so
 *.dylib
+kustomize
```

Comment:
This actually needs to be `/kustomize` otherwise it'll match things like `cmd/kustomize/main.go` making that a bit challenging to work on. 😄

Reply:
nice catch! i totally missed this 😅